src/app/footer/index.ts

```diff
@@ -1,6 +1,6 @@
-import { upgradeAdapter } from "../adapter";
-import { FooterComponent } from "./footer.component";
+//import { upgradeAdapter } from "../adapter";
+export { FooterComponent } from "./footer.component";
```

Comment:
Any reason to leave this commented in the final version instead of simply removing it?